### PR TITLE
fix(details card): 🐛 fix cost not showing during share

### PR DESCRIPTION
**Pull request by mcnaveen**
## Why this PR?
- fix cost not showing during the share
- fixes comma in SQL file

image

### CODE REVIEW

File: components/card/details.tsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export default function CardDetails(props: CardDetailsProps) {
 
   const share = async () => {
     const shareData = {
-      text: `${getCurrencySymbol()} ${subscription} per ${subscription.payment_cycle}`,
+      text: `${getCurrencySymbol()} ${subscription.cost} per ${subscription.payment_cycle}`,
```
Comment (gokulkrishh) [inline components/card/details.tsx:35]: Good catch, or i wrote like this so someone would contribute it. 👀

Comment (gokulkrishh) [reply components/card/details.tsx:35]: can we remove `per`? as its already saying monthly or yearly.

File: supabase/setup.sql [hunk 2]
```diff
@@ -11,7 +11,7 @@ CREATE TABLE users (
   avatar_url text not null,
   updated_at timestamp with time zone default current_timestamp,
   created_at timestamp with time zone default current_timestamp,
-  filter_by text not null default 'monthly'
+  filter_by text not null default 'monthly',
```
Comment (gokulkrishh) [inline supabase/setup.sql:14]: 👍

Comment (gokulkrishh) [reply supabase/setup.sql:14]: I need to actually remove this filter_by later as filters are local states. Thx.

**Review by gokulkrishh — CHANGES_REQUESTED** [review]

File: components/card/details.tsx (hunk 1 above)
Comment (mcnaveen) [reply components/card/details.tsx:35]: yeah, that make sense.

File: supabase/setup.sql (hunk 2 above)
Comment (mcnaveen) [reply supabase/setup.sql:14]: Cool

**Review by gokulkrishh — APPROVED** [review]

**Pull request merged**